x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/index.tsx

```diff
@@ -64,7 +64,7 @@ const {
   routes: customLogsRoutes,
 } = createWizardContext({
   initialState,
-  initialStep: 'selectLogs',
```

Comment:
Why is `selectLogs` no longer the initial step? it it a consequence of the client-side routing?

Reply:
No, it's just a requirement to be now the configureLogs one. I think we might go back to the selectLogs when those onboarding flows are ready